.github/workflows/Push.yaml

```diff
@@ -0,0 +1,137 @@
+name: libsovtoken Push
+
+on:
+  push:
+    branches:
+      - stable
```

Comment:
This should include `master` and `stable`.  Sorry if our conversation was not clear before.